understanding/22/accessible-authentication.html

```diff
@@ -52,6 +52,9 @@ <h2>Intent of Accessible Authentication</h2>
 
         <p>Copy and paste can be relied on to avoid transcription. Examples of this are password managers automatically filling in username and passwords, or web-based command line interfaces asking for a password that can be copied from a local source. Blocking people or programs from pasting into authentication fields, or using a different format between the copied text and the input field, would force the user to transcribe information and therefore fail this criterion.</p>
 
+        <p>If a <a href="https://www.w3.org/TR/turingtest/">CAPTCHA</a> is used as part of an authentication process, there must be a method that does not include a cognitive function test. If the test is based on something the website has set such as remembering or transcribing a word, or recognizing a picture the website provided, that would be a cognitive functional test. Recognizing common objects, or a picture the user has provided, would not be a cognitive functional test. Some forms of object recognition may require an understanding of a particular culture. For example, taxis can appear differently in different locales. This is an issue for many people, including people with disabilities, but it is not considered an accessibility-specific issue.</p>
+        <p>Some CAPTCHAs and cognitive function tests used for authentication may only appear in certain situations, such as when ad blockers or present or after repeated incorrect password entry. This criterion applies when these tests are used regardless of whether they are used every time or only triggered by specific scenarios.</p>
```

Comment:
"when ad blockers or present" > "when ad blockers *are* present"

i'd also sneak in a comma there after "present".

Reply:
Thanks, added.